### DISCUSSION TITLE
Can someone create a cask for LabPlot please?

### DISCUSSION THREAD

**Discussion by luzpaz** [Homebrew/discussions#4673] [Writing Formulae/Casks]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

Source: https://invent.kde.org/education/labplot
Website: https://labplot.kde.org/
macOS download image: https://download.kde.org/stable/labplot/labplot-2.10.1.dmg

Description: 
> LabPlot is a FREE, open source and cross-platform Data Visualization and Analysis software accessible to everyone.
>
> High-quality Data Visualization and Plotting with just a few clicks
> Reliable and easy Data Analysis and Statistics, no coding required!
> Intuitive and fast Computing with Interactive Notebooks
> Effortless Data Extraction from plots and support for Live Data
> Smooth Data Import and Export to and from multiple formats
> Available for Windows, macOS, Linux and FreeBSD

**Comment by SMillerDev** [comment]
> Is your favorite software missing from Homebrew? Then you’re the perfect person to resolve this problem.

https://docs.brew.sh/Adding-Software-to-Homebrew
